net/http/src/THttpServer.cxx

```diff
@@ -717,6 +717,11 @@ Int_t THttpServer::ProcessRequests()
       if (!arg)
          break;
 
+      if (arg->fFileName == "root_batch_holder.js") {
```

Comment:
Could you introduce a special syntax for the `ProcessBatchHolder()` filename instead of something that really looks like a filename? E.g. `$$$BATCHHOLDER$$$` or whatever :-)